aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/LastSyncMetadata.java

```diff
@@ -84,22 +95,25 @@ static <T extends Model> LastSyncMetadata deltaSyncedAt(@NonNull String modelCla
      */
     public static <T extends Model> LastSyncMetadata neverSynced(@NonNull String modelClassName) {
```

Comment:
This is a breaking change. Need to leave alone and created method overload. Internally, we should check for usages and make sure we are't using this method. Can mark this one as deprecated.